### DISCUSSION TITLE
Audio-Webui

### DISCUSSION THREAD

**Discussion by pajeet-dev** [gitmylo/audio-webui#123] [Q&A]
Is it possible to  install an existing rvc model file ??? if so where do you put it ??????

**Comment by gitmylo** [comment]
You put it in `data/models/rvc/name_of_model/`

So if you have a model named "test" and it has a file named "test.pth" and "added_13552.index" or something like that, you'll put it in a folder named "test", like this:

* data/models/rvc/`test`
  * test.pth
  * added_13552.index

the .index file is optional, but if the model comes with it, i really recommend it

**Comment by pajeet-dev** [comment]
Thank you for your help, your advice has got me up and running
